### PR TITLE
Fix undefined index warning. Closes #317

### LINKED ISSUE #317: Undefined index: np_relay_server_url
Kari just reported this on jeanniestriedandtruerecipes.com -- apparently it started March 29, so probably with NP Support 2.3.

`[15-Apr-2024 16:36:23 UTC] PHP Notice:  Undefined index: np_relay_server_url in /home/nginx/domains/jeanniestriedandtruerecipes.com/public/wp-content/plugins/blog-tutor-support/includes/class-support-helpers.php on line 414`

### PR DESCRIPTION
**Pull request by ecotechie**
@jchristopher I also changed the hard coded Relay URL for our helper method. Let me know if that's not okay and I can swap it back.

### CODE REVIEW

**Comment by ecotechie** [comment]
@blogtutor and @jchristopher good to merge?

**Review by 1337rout — APPROVED** [review]
LGTM

File: includes/class-support-snapshot.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public static function ping_relay() {
 		if ( isset( $_REQUEST['np_dispatch'] ) && isset( $_REQUEST['action' ] ) && 'trigger_snapshot' == $_REQUEST['action'] ) {
 			$options          = get_option( 'blog_tutor_support_settings' );
 			$site_api_key     = $options['np_relay_api_token'] ?? '';
-			$relay_server_url = ! empty( $options['np_relay_server_url'] ) ? $options['np_relay_server_url'] : 'https://relay.nerdpress.net';
+			$relay_server_url = ! empty( $options['np_relay_server_url'] ) ? $options['np_relay_server_url'] : NerdPress_Helpers::relay_server_default_url();
```
Comment (1337rout) [inline includes/class-support-snapshot.php:49]:
```suggestion
			$relay_server_url = ! empty( $options['np_relay_server_url'] ) ? $options['np_relay_server_url'] : rtrim( NerdPress_Helpers::relay_server_default_url(), '/' );
```

I sent  you a message about this, but we want to keep that trailing slash removed here so everything will behave exactly the same.

**Comment by blogtutor** [comment]
Tested on dev.nerdpress.net.  Couldn't reproduce the undefined index warning before, but I tested this branch and it's still sending snapshots back correctly. (empty setting, it goes to relay.nerdpress.net, and if something is set it goes to the correct location instead.)

**Pull request merged**